src/Bridge/Symfony/Resources/config/doctrine_orm.xml

```diff
@@ -78,7 +78,7 @@
                  public="true" />
 
         <service id="fidry_alice_data_fixtures.persistence.persister.doctrine.object_manager_persister"
-                 class="Fidry\AliceDataFixtures\Bridge\Doctrine\Persister\ObjectManagerPersister"
+                 class="Fidry\AliceDataFixtures\Bridge\Doctrine\Persister\ManagerRegistryPersister"
```

Comment:
`ObjectManagerPersister` take an EntityManager as parameter, but we give a Registy

Reply:
it's curious that this works...